src/scripts/ps1/replicate/RunReplicateEvents-Dataflow.ps1

```diff
@@ -39,9 +39,14 @@
 #---------------------------------------------------------------------
 # The job config file.
 #---------------------------------------------------------------------
-$jobConfigFile = $gcpBucketPrefix + '-test/config/replicate/job-config-events-replication.toml'
+#$jobConfigFile = $gcpBucketPrefix + '-test/config/replicate/job-config-events-replication.toml'
 
-mvn compile exec:java -D exec.mainClass=com.cognite.sa.beam.replicate.ReplicateEvents -D exec.args="--cdfInputSecret=$cdfInputSecret --cdfInputHost=$cdfInputHost --cdfOutputSecret=$cdfOutputSecret --cdfOutputHost=$cdfOutputHost --jobConfigFile=$jobConfigFile --fullRead=$fullRead --project=$gcpProject --runner=DataFlowRunner --gcpTempLocation=$gcpBucketPrefix-test/temp --stagingLocation=$gcpBucketPrefix-test/stage/replicate/replicate-events --region=europe-west1 --experiments=shuffle_mode=service --numWorkers=4 --maxNumWorkers=6 --experiments=enable_stackdriver_agent_metrics --workerMachineType=e2-standard-2"
+#---------------------------------------------------------------------
+# The delta identifier
+#---------------------------------------------------------------------
+#$deltaIdentifier = 'event-replicator'
```

Comment:
Should we enable this variable (i.e. uncomment it). Then it becomes more explicit that it has to be set.